<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetBuilder</class>
 <widget class="QMainWindow" name="DatasetBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DatasetBuilder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget
{
	background-color:#121212
}

QWidget
{
	font: &quot;Segoe UI&quot;;
}

QMainWindow
{
	background-color:#121212
}

QPushButton
{
	background-color:#ffffff;
	border-style: outset;
	border-width: 2px;
    border-radius: 3px;
	border-color:  #ffffff;
	font: bold;
	padding: 0.2em;
}

QPushButton:pressed
{
	background-color:#ccccccc;
}

QMenuBar {
    background-color: #272727;
    color: #ffffff;
}

QMenuBar::item:selected {
    background: #101010;
}

QMenu {
    background-color: #272727;
    color: #ffffff;
}

QMenu::item:selected {
    background-color: #151515;
    color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QListWidget" name="m_lwRemainingFolders">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget
{
	background-color: #242424;
	color: #ffffff;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="m_lblImg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	color: #ffffff;
	font: 20px bold;
}</string>
        </property>
        <property name="text">
         <string>No image yet</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="m_buttonWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#m_buttonWidget
{
	background-color:#272727
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_textFiledTitle">
         <property name="styleSheet">
          <string notr="true">QLabel
{
	color: #ffffff;
	font: 12px bold;
}</string>
         </property>
         <property name="text">
          <string>Enter translation:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_leTextImg">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbSkip">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Skip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="m_pbSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1049</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="m_acInputFolders"/>
    <addaction name="m_acOutputFolder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="m_acInputFolders">
   <property name="text">
    <string>Select input folders</string>
   </property>
  </action>
  <action name="m_acOutputFolder">
   <property name="text">
    <string>Select output folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbSkip</sender>
   <signal>clicked()</signal>
   <receiver>DatasetBuilder</receiver>
   <slot>SkipCurrentImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1038</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>686</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pbSave</sender>
   <signal>clicked()</signal>
   <receiver>DatasetBuilder</receiver>
   <slot>Save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>981</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>734</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SkipCurrentImage()</slot>
  <slot>Save()</slot>
 </slots>
</ui>
